From 5e9310bbbcd5086ea9a3edf85d523c4c2a57f1c3 Mon Sep 17 00:00:00 2001
From: Andrew Beekhof <andrew@beekhof.net>
Date: Tue, 8 Dec 2015 13:54:12 +1100
Subject: [PATCH] NovaEvacuate should invoke fence_compute with action 'off'

Conceptually we are resurrecting in one direction only (off) and not
bringing it back to the current host afterwards (on)

Also it will overwrite the attrd variable too soon.

Change-Id: I9694945ca7eedae4f5cb6758fe1e8ce7f72ae808
---
 ocf/NovaEvacuate | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/heartbeat/NovaEvacuate b/heartbeat/NovaEvacuate
index a17a159..0e22d7e 100644
--- a/heartbeat/NovaEvacuate
+++ b/heartbeat/NovaEvacuate
@@ -198,7 +198,7 @@ handle_evacuations() {
 		return $OCF_SUCCESS
 	    fi
 
-	    fence_compute ${fence_options} -o reboot -n $node
+	    fence_compute ${fence_options} -o off -n $node
 	    rc=$?
 
 	    if [ $rc = 0 ]; then
-- 
1.9.1

